src/libraries/Microsoft.Extensions.FileSystemGlobbing/ref/Microsoft.Extensions.FileSystemGlobbing.cs

```diff
@@ -150,7 +150,7 @@ public partial class LiteralPathSegment : Microsoft.Extensions.FileSystemGlobbin
         public LiteralPathSegment(string value, System.StringComparison comparisonType) { }
         public bool CanProduceStem { get { throw null; } }
         public string Value { get { throw null; } }
-        public override bool Equals(object obj) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
         public bool Match(string value) { throw null; }
     }
```

Comment:
Do we need to update `WildcardPathSegment` with ArgNull checks? It looks like they can't accept `null`, but nothing is guaranteeing someone doesn't create an instance with `null`.